### ISSUE TITLE
Configmaps are always created in a default namespace

### ISSUE THREAD

**Issue by mateusz-blaszkowski** [stackanetes/stackanetes#48]
See `stackanetes/common/utils.py` in PR #42. Results in PODs not created properly:

```
Events:
  FirstSeen     LastSeen        Count   From                    SubobjectPath   Type            Reason          Message
  ---------     --------        -----   ----                    -------------   --------        ------          -------
  18m           18m             1       {default-scheduler }                    Normal          Scheduled       Successfully assigned searchlight-init-mdavi to 10.3.9.6
  18m           8s              90      {kubelet 10.3.9.6}                      Warning         FailedMount     Unable to mount volumes for pod "searchlight-init-mdavi_openstack-system(0731e1f9-44d2-11e6-85f6-f8bc1206b7d0)": configmaps "searchlight-init" not found
  18m           8s              90      {kubelet 10.3.9.6}                      Warning         FailedSync      Error syncing pod, skipping: configmaps "searchlight-init" not found

```

**Comment by PiotrProkop** [comment]
Is this issue resolved in #49 ? @mateusz-blaszkowski

**Comment by mateusz-blaszkowski** [comment]
Yes, resolved in #49
